src/openrct2/ride/TrackPaint.h

```diff
@@ -496,6 +497,18 @@ void TrackPaintUtilOnridePhotoSmallPaint(
     PaintSession& session, Direction direction, int32_t height, const TrackElement& trackElement);
 void TrackPaintUtilOnridePhotoPaint(
     PaintSession& session, Direction direction, int32_t height, const TrackElement& trackElement);
+void TrackPaintUtilOnridePhotoPaint2(
+    PaintSession& session, Direction direction, int32_t height, int32_t trackHeightOffset, int32_t supportsAboveHeightOffset,
+    const TrackElement& trackElement);
+inline void TrackPaintUtilOnridePhotoPaint2(
+    PaintSession& session, Direction direction, const TrackElement& trackElement, int32_t height,
+    int32_t supportsAboveHeightOffset = kGeneralSupportHeightOnRidePhoto, int32_t trackHeightOffset = 3)
```

Comment:
Any reason for making a `trackHeightOffset` a parameter? Seems like no call ever did provide a value different from 3.

Reply:
_Most_ don’t - but the Mine Train uses 2 and the River Rapids use 13.

Reply:
Argh, my bad then.